frontend/passkeys.js

```diff
@@ -120,34 +115,42 @@ function getPasskeyNameFromUser() {
         const cancelButton = modalElement.querySelector('#cancel-button');
         const input = modalElement.querySelector('#passkey-name');
         const errorMessage = modalElement.querySelector('#passkey-name-error');
+        // Regex to allow only alphanumeric characters and spaces
+        const alphanumericRegex = /^[a-zA-Z0-9 ]*$/;
```

Comment:
Do we want to also allow characters like '-', '_'?  I can't think of any others.
Ignore this if you think it's unnecessary.

Reply:
I was wondering that too,. I'll add those and also limit the length.

Reply:
I've addressed these in https://github.com/guardian/janus-app/pull/569/commits/361d61433db7e022a69c7366cd5a4902efe385a0